edx_proctoring/tests/test_views.py

```diff
@@ -2,14 +2,18 @@
 """
 All tests for the proctored_exams.py
 """
+
+from __future__ import absolute_import
+
+from datetime import datetime, timedelta
 import json
-import pytz
+from string import Template  # pylint: disable=deprecated-module
```

Comment:
Is there a way of converting this to not using a deprecated module? It just makes me nervous about not going back to clean it up later.